cmd/importaccount.go

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2018-2020 The NATS Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/nats-io/jwt"
+	"github.com/nats-io/nkeys"
+	"github.com/nats-io/nsc/cmd/store"
+	"github.com/spf13/cobra"
+)
+
+func createImportAccountCmd() *cobra.Command {
+	var params ImportAccount
+	cmd := &cobra.Command{
+		Use:          "account --jwt <account-jwt>",
+		Short:        "Imports an account from a jwt file and resign with operator if self signed",
+		Example:      `nsc import account --jwt <account-jwt>`,
+		Args:         MaxArgs(0),
+		SilenceUsage: false,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := RunMaybeStorelessAction(cmd, args, &params); err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+	cmd.Flags().BoolVarP(&params.skip, "skip", "", false, "skip validation issues, they can be edited out")
+	cmd.Flags().BoolVarP(&params.overwrite, "overwrite", "", false, "overwrite existing account")
+	cmd.Flags().StringVarP(&params.file, "jwt", "j", "", "account jwt to import")
+	cmd.MarkFlagRequired("jwt")
+	return cmd
+}
+
+func init() {
+	importCmd.AddCommand(createImportAccountCmd())
+}
+
+type fileImport struct {
+	file      string
+	skip      bool
+	overwrite bool
+}
+
+type ImportAccount struct {
+	SignerParams
+	fileImport
+}
+
+func (p *ImportAccount) SetDefaults(ctx ActionCtx) error {
+	p.SignerParams.SetDefaults(nkeys.PrefixByteOperator, true, ctx)
+	return nil
+}
+
+func (p *ImportAccount) PreInteractive(ctx ActionCtx) error {
```

Comment:
Add field to ask for file by prompt here (that means that file is no longer required in the flag, so need to verify in Validate)

Reply:
will not implement